site/src/sources/kolibri.json

```diff
@@ -23,26 +23,13 @@
       ]
     },
     {
-      "name": "Accordion",
-      "url": "https://public-ui.github.io/docs/components/accordion",
```

Comment:
I probably manually added this `url` field as well (and also for Details) -- but it would be good to keep it, and probably also to generate similar `url` fields for the other widgets so that they show up properly in the generated pages.

Reply:
Thank you @dbaron for your hint. I have added all urls.